jans-cedarling/cedarling/src/bootstrap_config/jwt_config.rs

```
@@ -195,3 +196,39 @@ impl JwtConfig {
         self
     }
 }
+
+/// Raw JWT config
+pub struct JwtConfigRaw {
+    /// JWKS
+    pub jwks: Option<String>,
+    /// JWT signature validation
+    pub jwt_sig_validation: bool,
+    /// JWT status validation
+    pub jwt_status_validation: bool,
+    /// Supported signature algorithms
+    pub signature_algorithms_supported: Vec<String>,
+}
+
+impl From<JwtConfigRaw> for JwtConfig {
+    fn from(raw: JwtConfigRaw) -> Self {
+        Self {
+            jwks: raw.jwks,
+            jwt_sig_validation: raw.jwt_sig_validation,
+            jwt_status_validation: raw.jwt_status_validation,
+            signature_algorithms_supported: raw.signature_algorithms_supported
+                .into_iter()
+                .filter_map(|alg| match alg.as_str() {
```

unsupported algorithm's are silently ignored without warning also missing many supported algorithms(PS256,PS384,PS512,EdDSA)